src/routes/commentRoutes.ts

```diff
@@ -1,59 +1,59 @@
 import { Hono } from 'hono';
 import { validateId } from '../middleware/idValidation';
-// import { jwt } from 'hono/jwt';
-// import { isSignedIn } from '../middleware/authValidation';
+import { jwt } from 'hono/jwt';
+import { isSignedIn } from '../middleware/authValidation';
 import { commentController } from '../controllers/commentController';
 import { validateComment } from '../middleware/commentValidation';
 
-// const JWT_ACCESS_SECRET = process.env.JWT_ACCESS_SECRET!;
+const JWT_ACCESS_SECRET = process.env.JWT_ACCESS_SECRET!;
```

Comment:
we can use rest here const { JWT_ACCESS_SECRET } = process.env;

Reply:
if i'll do like this i will need to use ! operator in every jwt middleware   
```
jwt({
    secret: JWT_ACCESS_SECRET!,
  }),
```
maybe it will be more elegant to use ! operator in one place instead of many?